src/main/java/org/opensearch/sdk/SDKClusterService.java

```diff
@@ -42,6 +43,15 @@ public ClusterState state() {
         return extensionsRunner.sendClusterStateRequest(extensionsRunner.getExtensionTransportService());
     }
 
+    /**
+     *  Returns the local extension node
+     *
+     * @return the local extension node
+     */
+    public DiscoveryExtensionNode localNode() {
```

Comment:
We should come up with a consistent policy on when to use record-style getters like this vs. `getLocalNode()` style. 

In general, record-style is appropriate for immutable return values.